.github/workflows/detect-breaking-changes-build.yml

```diff
@@ -105,10 +111,10 @@ jobs:
           name: buildBase
       
       - name: Unzip artifact from pr
-        run: unzip pr_built_packages.zip -d ./pr && rm pr_built_packages.zip
+        run: unzip -j pr_built_packages.zip -d ./pr && rm pr_built_packages.zip
```

Comment:
**Question:** what does exactly the `junk paths` option do here?

Reply:
Exactly. Without `-j` zip unpacks each tarball into `./pr/packages/grafana-<package_name>/@<package_name>.tar` which felt a bit redundant.

However thinking about it now if we kept this directory structure we might be able to simplify the changes to the shell script. 🤔

Reply:
Aaa, I see, thanks! 

I would say it's probably better as it is for now (even if we need a little bit more logic in the shell script), as for me at least the cleaner paths make it easier to reason about the whole thing. 👍